src/css/docs-ndl.css

```diff
@@ -332,41 +332,19 @@ body.docs-ndl .lists .sect2 {
   padding: 1rem;
   /* margin: 0 1rem 1rem 0; */
   background: var(--color-white);
-  border-radius: 0.5rem;
-  /* box-shadow: 0 4px 6px -1px rgba(0, 0, 0, 0.1), 0 2px 4px -1px rgba(0, 0, 0, 0.06); */
-  /* border: 1px solid #d5d9de; */
-}
-
-body.docs-ndl.explainer .cards .sect2 a {
-  background: none;
 }
 
-body.docs-ndl.explainer .cards.selectable .sect2:hover {
-  transform: none;
+body.docs-ndl .cards .sect2 > a,
+body.docs-ndl .cards:not(.selectable) .sect2,
+body.docs-ndl .widget:not(.video) {
+  /* box-shadow: 0 2px 4px -1px rgba(0, 0, 0, 0.1), 0 1px 2px -1px rgba(0, 0, 0, 0.06); */
+  border: 1px solid var(--panel-border-color);
+  border-radius: 0.5rem;
 }
 
 body.docs-ndl.color-docs .sect2 {
   color: rgba(var(--colors-neutral-10));
   background: var(--color-docs);
-  /* background: none; */
-  /* box-shadow: 0 4px 6px -1px rgba(0, 0, 0, 0.1), 0 2px 4px -1px rgba(0, 0, 0, 0.06); */
-  /* border: 1px solid #d5d9de; */
-  /* padding: 0; */
-}
-
-body.docs-ndl.explainer .cards .sect2 a .icon {
-  float: left;
-  position: absolute;
-}
-
-/* body.docs-ndl.explainer .cards .sect2 .icon img {
-  height: 4rem;
-  width: 4rem;
-} */
-
-body.docs-ndl.explainer .cards .sect2 a h3,
-body.docs-ndl.explainer .cards .sect2 a .description {
-  margin-left: 5rem;
 }
 
 body.docs-ndl.color-docs .cards .sect2 h3,
```

Comment:
In line 412, could we have something like this for the icons to be aligned to the left of the text? I suppose this is for the list widget that is being used in the Tutorials subpage? 

```
body.docs-ndl .lists .sect2 .icon img,
body.docs-ndl .icon-s .sect2 .icon img {
  width: 3rem;
  height: 3rem;
  padding: 0;
  margin: 40px;
  float: left
}
```